### ISSUE TITLE
I would love to try this at our small studio

### ISSUE THREAD

**Issue by Shahar0z13** [LOOKSFilm/Metadata-Kumpel#1]
i suppose its probably developed internally by you. will you be able to share the package "EditShareAPI" with me? i cant find this package anywhere.

im currently using excel to create custom sidecar xmls so i would really love to try this out.

**Comment by LOOKSFilm** [comment]
Yes this is build for a specific workflow we use internally. Never the less it might work on your setup too. I added a IP input field on the login screen. There are some functions e.g. "renaming" that might not work, because i hardcoded some metadatafield names there. 

I also switched the EditShareAPI repo to public. It's also written by me. There is an Module on the EditShare server as well, but there is no documentation from EditShare, so i thought it's faster to rewrite it, than reverse engineer it...
I've no motivation to write an Documentation for the module either (since i am the only one who uses it atm). It only handles some requests of the [EditShare API](https://developers.editshare.com/) endpoints (by far not all). I am sometimes adding a few, but only when i need them.

**Comment by Shahar0z13** [comment]
Thank you so much!!

i will play with it a bit and let you know
i really appreciate the help, its really hard finding resources online for editshare. they really lack in documentation.

so thanks again! and sorry if im doing anything wrong in here, im not accustomed to github etiquette :)
